app/templates/user_homepage.html

```diff
@@ -61,6 +61,35 @@ <h2>Ustawienia konta</h2>
         function logout(){
             window.location.href = "{{ url_for('logout') }}";
         }
+
+        
+    function askForLocation() {
+        navigator.geolocation.watchPosition(function(position) {
+            var data = {
+                latitude: position.coords.latitude,
+                longitude: position.coords.longitude
+            }
+
+            fetch('http://127.0.0.1:5000/user_homepage', {

```

Comment:
not user but 
```suggestion
            /user_homepage', {
```

It's a guess but it should also work